src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,10 +1,60 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+export type NewMovieProps = {
+  onAdd: (movie: Movie) => void
+};
+
+export const NewMovie = ({ onAdd }: NewMovieProps) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState<string>('');
+  const [description, setDescription] = useState<string>('');
+  const [imgUrl, setImgUrl] = useState<string>('');
+  const [imdbUrl, setImdbUrl] = useState<string>('');
+  const [imdbId, setImdbId] = useState<string>('');
```

Comment:
Create only one state for movie
```suggestion
  const [movie, setMovie] = useState({
    movie: '',
    description: '',
    .......
  });```

Reply:
Creting one movie object mean passing: 
        onChange={(value) => setMovie((prev) => ({ ...prev, title: value }))}
or creating extra functions, and it is a lot of usless code